notebook/services/kernels/handlers.py

```diff
@@ -452,7 +452,6 @@ def on_close(self):
                 # close the socket directly, don't wait for the stream
```

Comment:
This comment should probably be removed as misleading. And the assignment below is now unnecessary.

Reply:
Good atch, fixed.